sdcm/sct_config.py

```diff
@@ -1222,6 +1222,10 @@ class SCTConfiguration(dict):
         dict(name="availability_zone", env="SCT_AVAILABILITY_ZONE",
              type=str,
              help="Availability zone to use. Same for multi-region scenario."),
+        dict(name="aws_auto_availability_zone", env="SCT_AWS_AUTO_AVAILABILITY_ZONE",
+             type=boolean,
+             help="Not limiting availability zones, in order for the cloud provider to automatically select"
```

Comment:
please update the description, and rename `aws_auto_availability_zone`, it's a fallback similar to `instance_provision_fallback_on_demand`

so something like `aws_fallback_to_next_availability_zone` might be clearer.

Reply:
fixed